test/safe/discovery.test.ts

```diff
@@ -58,6 +109,7 @@ describe("Discovery Queue Test", () => {
 		expect(states.length).toBe(2);
 	}
 
+	// eslint-disable-next-line jest/expect-expect
```

Comment:
What does this rule do? Seems it wasn't needed before

Reply:
These two tests don't have assertions directly inside the' it' block since they are hidden away inside a seperate function.
It's one of the 250 warning messages that get thrown on commit.

Alternatively we could move the assertion inside the it blocks, its more code but its much more obvious for testing.